.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: CI
```

Comment:
this is already in master. can you clean up your PR with only your change. THanks